exchanges/bitstamp/bitstamp_websocket.go

```diff
@@ -107,8 +117,19 @@ func (b *Bitstamp) wsHandleData(respRaw []byte) error {
 		if err != nil {
 			return err
 		}
-		currencyPair := strings.Split(wsResponse.Channel, currency.UnderscoreDelimiter)
-		p, err := currency.NewPairFromString(strings.ToUpper(currencyPair[2]))
+
+		currencyPair := strings.Split(wsResponse.Channel, currency.UnderscoreDelimiter)[2]
```

Comment:
It would be a nice minor improvement to have a len check first in the event that this gets changed and we err instead of panic